tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java

```diff
@@ -356,7 +325,7 @@ private void deleteTopicsByKafkaAdmin(AdminClient admin, Set<String> topicsToDel
     @Test(timeOut = 10000)
     public void testCreateAndDeleteTopics() throws Exception {
         Properties props = new Properties();
-        props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:" + getClientPort());
+        props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:" + getKafkaBrokerPort());
```

Comment:
this should stay getClientPort, because when we run the tests using the proxy this becomes the proxy port